tabs/tab_data.py

```
@@ -3,75 +3,84 @@
+    df_matched: reactive.Value[Optional[pd.DataFrame]], 
+    is_matched: reactive.Value[bool], 
+    matched_treatment_col: reactive.Value[Optional[str]], 
+    matched_covariates: reactive.Value[List[str]]
+) -> None:
+
     # ไม่ต้องประกาศ ns = session.ns
     # ใช้ input.id() ได้เลย (Shiny ตัด prefix ให้เองใน Module)
-    
-    is_loading_data = reactive.Value(False)
+
+    is_loading_data: reactive.Value[bool] = reactive.Value(False)
```

Ruff (0.14.10)

Boolean positional value in function call

(FBT003)